go/vt/vttablet/onlineddl/executor.go

```diff
@@ -2863,6 +2863,17 @@ func (e *Executor) executeCreateDDLActionMigration(ctx context.Context, onlineDD
 			}
 		}
 	}
+	if originalCreateTable, ok := ddlStmt.(*sqlparser.CreateTable); ok {
+		newCreateTable := sqlparser.CloneRefOfCreateTable(originalCreateTable)
+		// Rewrite this CREATE TABLE statement such that CONSTRAINT names are edited,
+		// specifically removing any <tablename> prefix.
+		if _, err := e.validateAndEditCreateTableStatement(ctx, onlineDDL, newCreateTable); err != nil {
```

Comment:
unrelated to this PR, but why do we pass a context to this function? Functions that don't perform RPCs do not need to take in a context.

Reply:
This specific function indeed doesn't use the given `context`. There's many other non-gRPC functions that do use a `context` because they either use a `Conn`ection (context cancellation ensures connection terminates) or loops/waits.